Update target.md (#4712)
fishtown-analytics/docs.getdbt.com: website/docs/reference/dbt-jinja-functions/target.md

---
title: "About target variables"
sidebar_label: "target"
id: "target"

> description: "The `target` variable Contains information about your connection to the warehouse."

Reviewer: ```suggestion
description: "The `target` variable contains information about your connection to the warehouse."
```

---

The `target` variable contains information about your connection to the warehouse.

- **dbt Core:** These values are based on the target defined in your [profiles.yml](/docs/core/connect-data-platform/profiles.yml) file. Please note that for certain adapters, additional configuration steps may be required. Refer to the [set up page](/docs/core/connect-data-platform/about-core-connections) for your data platform.

> - **dbt Cloud** To learn more about setting up your adapter in dbt Cloud, refer to the [About data platform connections](/docs/cloud/connect-data-platform/about-connections).

Author: ```suggestion
- **dbt Cloud** To learn more about setting up your adapter in dbt Cloud, refer to [About data platform connections](/docs/cloud/connect-data-platform/about-connections).
```

>    - **dbt Cloud Scheduler:** `target.name` is defined per job as described in [Custom target names](/docs/build/custom-target-names). For other attributes, values are defined by the deployment connection. To check these values, click **Deploy** and select **Environments**. Then, select the relevant deployment environment, and click **Settings**.

Reviewer: might be good to link to the scheduler
```suggestion
   - **[dbt Cloud Scheduler](/docs/deploy/job-scheduler)**: `target.name` is defined per job as described in [Custom target names](/docs/build/custom-target-names). For other attributes, values are defined by the deployment connection. To check these values, click **Deploy** and select **Environments**. Then, select the relevant deployment environment, and click **Settings**.
```

>    - **dbt Cloud IDE:** These values are defined by your connection and credentials. To edit these values, click the cog in the top right, select **Profile settings**, and click **Credentials**. Select and edit a project to set up the credentials and target name.

Reviewer: adding link to ide docs and also i used to use 'cog' and was been given feedback that we use the term 'gear icon' instead.

```suggestion
   - **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)**: These values are defined by your connection and credentials. To edit these values, click the gear icon in the top right, select **Profile settings**, and click **Credentials**. Select and edit a project to set up the credentials and target name.
```

Reviewer: also might be helpful to have screenshots for scheduler and ide but not a blocker or urgent
